bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/cmd/AntiPatternCommandParser.java

```diff
@@ -141,10 +152,29 @@ public Options getOptions() {
             Option.builder(REWRITE_SQL_FLAG_NAME)
                     .argName(REWRITE_SQL_FLAG_NAME)
                     .required(false)
-                    .desc("flag specifying if the queries should be rwwritten using an LLM")
+                    .desc("flag specifying if the queries should be rewritten using an LLM")
                     .build();
     options.addOption(rewriteSQLFlag);
 
+    Option llmStrictValidation =
+            Option.builder(LLM_STRICT_VALIDATION_FLAG_NAME)
+                    .argName(LLM_STRICT_VALIDATION_FLAG_NAME)
+                    .required(false)
+                    .desc("flag specifying that LLM results will be discarded if the validation fails (Syntactically " +
+                            "invalid or antipattern still present)")
+                    .build();
+    options.addOption(llmStrictValidation);
+
+    Option llmRetriesSQLFlag =
+            Option.builder(LLM_RETRIES_FLAG_NAME)
+                    .argName(LLM_RETRIES_FLAG_NAME)
+                    .hasArg()
+                    .required(false)
+                    .desc("flag specifying the number of retries when the LLM generates an incorrect SQL " +
+                            "query (Syntactically invalid or antipattern still present)")
+                    .build();
+    options.addOption(llmRetriesSQLFlag);
```

Comment:
no clear to me what the default value of this argument will be. what happens if it is not specified, probably i am missing something. will test

Reply:
anyhow let's please make sure that the default is 0.
for backwards compatibility we want to keep default behavior as is. Only if user spacifies llmRetries>0 or llmStrictValidation then we apply any validation logic

Reply:
The default value is not 0, the value is in the get function for this argument

Reply:
Done